src/devices_models/devices/hybrid_generation.jl

```diff
@@ -167,6 +167,113 @@ get_min_max_limits(
     ::Type{<:AbstractHybridFormulation},
 ) = PSY.get_state_of_charge_limits(PSY.get_storage(device))
 
+# TODO: Refactor these constraints to be used only by HybridSystems methods and change name
+# of the function
+
+function _add_lower_bound_range_constraints_impl!(
```

Comment:
@sourabhdalvi this code also needs to be refactored. This code should be implemented only here.